index.html

```diff
@@ -13,6 +13,16 @@
 
 <body>
     <div class="container">
```

Comment:
On a tags that aren't used yet, you can add the  href attribute to them still with a # so it just points to the current page, again, not necessary, but you will commonly see it